corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -49,14 +56,73 @@
               .stacked(true)
             ;
             chart.yAxis.tickFormat(d3.format(',.0f'));
-            d3.select('#chart svg')
+            chart.color(["#004ebc", "#e53e30"]);
+            d3.select('#perform_chart svg')
                 .datum([performingSeries, activeNonPerformingSeries])
                 .call(chart);
             nv.utils.windowResize(chart.update);
             return chart;
         });
     }
 
+    function parseToUTCDate(month){
```

Comment:
@benrudolph Oh cool, I can definitely take a look at momentjs! I was working with the JS Date objects and dealing with timezone issues, but momenjs will be super helpful